sherpa/data.py

```diff
@@ -1799,7 +1819,7 @@ def html_data1dint(data):
     if data.syserror is not None:
         meta.append(('Systematic error', data.syserror))
 
-    ls = [formatting.html_section(meta, summary=dtype + ' Summary',
+    ls = [formatting.html_section(meta, summary=f'{dtype} Summary',
```

Comment:
as above. Again, I think getting the dtype string here instead of 21 lines above improves readability (based on my  experience when I tried to do changes to this ago a while back)